src/test/java/Pages/MicrosoftLoginPage.java

```diff
@@ -42,11 +46,13 @@ public boolean isLoaded() {
 
     public void fillForm() throws Exception {
         emailInput.clear();
-        emailInput.sendKeys("");
+        emailInput.sendKeys("armando.cifuentes@itexico.net");
         nextButton.click();
-        Thread.sleep(1000);
+
+        wait.until(ExpectedConditions.visibilityOf(passwordInput));
+
         passwordInput.clear();
-        passwordInput.sendKeys("");
+        passwordInput.sendKeys("Intheend12");
```

Comment:
Para los passwords debes agregar una funcion para hacer 'Salt' a los password, nuevamente aqui no importa por el tamano del proyecto pero en otras ocasiones si es importante hacerlo. Mas info:

https://www.novixys.com/blog/aes-encryption-decryption-password-java/#6_Decrypting_the_File

Reply:
Solved temporally. I'll read the article to encrypt the pass.